JobConfig/mixing/epilog.fcl

```diff
@@ -1,38 +1,36 @@
-# add primary and mixers to the list of step producers.  Note the StrawDigi maker uses a general selector if no module is specified
-physics.producers.makeSD.StrawGasStepModule : ""
+# mixing requires the SimEfficiency tables
+services.ProditionsService.simbookkeeper.useDb: true
 # configure the mixin efficiencies
 physics.filters.EleBeamFlashMixer.mu2e.simStageEfficiencyTags: ["EleBeamCat", "EleBeamFlashCat" ]
 physics.filters.MuBeamFlashMixer.mu2e.simStageEfficiencyTags: ["MuBeamCat", "MuBeamFlashCat" ]
 physics.filters.NeutralsFlashMixer.mu2e.simStageEfficiencyTags: ["NeutralsCat", "NeutralsFlashCat" ]
 physics.filters.MuStopPileupMixer.mu2e.simStageEfficiencyTags: ["MuBeamCat", "MuminusStopsCat", "MuStopPileupCat"]
 physics.filters.MuStopPileupMixer.mu2e.meanEventsPerPOTFactors : [ @local::Pileup.MuminusTargetStopPrescale ]
+# adjust detstep lists to include pileup
+physics.producers.makeSD.StrawGasStepModule : ""
+physics.producers.compressDigiMCs.simParticleTags : [compressDetStepMCs, @sequence::Mixing.StepMixSequence ]
+physics.producers.compressDigiMCs.caloShowerStepTags : [compressDetStepMCs, @sequence::Mixing.StepMixSequence ]
+physics.producers.CaloShowerROMaker.caloShowerStepCollection : [compressDetStepMCs, @sequence::Mixing.StepMixSequence ]
+physics.producers.CrvPhotons.crvStepModuleLabels : [ compressDetStepMCs, @sequence::Mixing.StepMixSequence ]
+physics.producers.CrvPhotons.crvStepProcessNames : [ "","","","","" ]
 #
-#  Only produce physics streams for now
+#  Physics signal and calibration streams, filtered on different triggers
 #
 physics.trigger_paths[0] : MixPath
 physics.trigger_paths[1] : Untriggered
-physics.EndPath : [ @sequence::Digitize.EndSequence, TriggeredOutput, UntriggeredOutput ]
 physics.end_paths : [ EndPath ]
-outputs : {
-  TriggeredOutput :  {
-    @table::Digitize.TriggeredOutput
-    fileName: "dig.owner.MixTriggered.version.sequencer.art"
-  }
-  UntriggeredOutput : {
-    @table::Digitize.UntriggeredOutput
-    fileName: "dig.owner.MixUntriggered.version.sequencer.art"
-  }
-}
+outputs : @local::Digitize.Outputs
+#  SignalOutput :  {
```

Comment:
Is there a reason why you need to keep these lines commented out?

Reply:
No reason.  I removed them